### PR TITLE
Updated Turkish translation.

### PR DESCRIPTION
**Pull request by AuroPick**
Updated Turkish translation and fixed some type errors.

### CODE REVIEW

**Comment by kaanhangithub** [comment]
I have already made a pull request  #757 about Turkish translation.

File: kofta/public/locales/tr/translation.json [hunk 1]
```diff
@@ -77,9 +77,8 @@
 		"viewUser": {
 			"editProfile": "profili düzenle",
 			"followsYou": "seni takip ediyor",
-			"folowers": "takipçiler",
```
Comment (AuroPick) [inline kofta/public/locales/tr/translation.json:80]: Fixed type error

**Comment by AuroPick** [comment]
> I have already made a pull request #757 about Turkish translation.

If you look up staging branch you can see your changes already made someone else.

**Comment by kaanhangithub** [comment]
> > I have already made a pull request #757 about Turkish translation.
> 
> If you look up staging branch you can see your changes already made someone else.

I pulled from new-design, my bad 😅

**Pull request merged**